explainer.md

```diff
@@ -850,11 +850,13 @@ Using existing manifest_url/start_url would be bad because there are multiple us
 
 With the primary goal of ensuring having a stable identifier to support various use cases of updating app's metadata. Using **start_url\_origin + id** is preferred as this is the most stable option.
 
-For the default field to use, **start_url** is preferred.
+For the default field to use, **start_url** is preferred. A fallback default id is supported to allow smooth transition from existing platforms where an id field doesn't exist in the manifest. Therefore when considering which default field to choose, being backward compatible is the key consideration.
 
-Because of the existing sync design of desktop PWAs, old chromes will be unavoidable to have duplicate apps if the default is not start_url, there might be weird behaviors when duplicate apps exist.
+Given that, we are comparing using either manifest_url or start_url as the default:
```

Comment:
maybe:

**start_url**  is preferred as the default **global_id** because:
  * it matches the exiting behavior of two implementations (Chromium Desktop & Firefox Mobile), allowing easy transition for the codebase of those browsers & developer expectations.
  * Chromium Android (and other manifest_url -based designs) have to migrate anyways due to it not supporting manifest_url data urls,
  * Since the **id** field can be recommended & eventually required for installability, this default doesn't matter as much once all manifests specify an **id**.